src/Files.App/Actions/Content/Tags/OpenAllTaggedActions.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) 2023 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+namespace Files.App.Actions
+{
+    sealed class OpenAllTaggedActions: ObservableObject, IAction
+    {
+		private readonly IContentPageContext _pageContext;
+
+		private readonly ITagsContext _tagsContext;
+
+		public string Label
+			=> "OpenAllItems".GetLocalizedResource();
```

Comment:
The label should be `OpenAllTaggedItems`

Reply:
I thought this as well but then I realized this string might be useful outside of tags.

Reply:
Should I revert it then?